### PR TITLE
removed warnings by adding key attributes

### LINKED ISSUE #6393: Fix 5 Key prop warn in MesheryAdapterPlayComponent.js
#### Current Behavior

Fix 5 Key prop warn in meshery/ui/components/MesheryAdapterPlayComponent.js :

All the map elements should have unique "key" prop: reference: https://stackoverflow.com/questions/28329382/understanding-unique-keys-for-array-children-in-react-js

Run `npm run lint` to check the warnings and fix them , all others have been fixed .

#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by sam1code**
Signed-off-by: Soumyakanta Pattanaik <samcse95@gmail.com>

**Description**

This PR fixes #6393

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by sam1code** [comment]
Also removed key warning from other 2 files too.

**Comment by leecalcote** [comment]
Thank you, @sam1code

File: ui/components/MesherySMIResults.js [hunk 1]
```diff
@@ -227,7 +227,7 @@ class MesherySMIResults extends Component {
                     </TableHead>
                     <TableBody>
                       {data?.map((row) => (
-                        <TableRow >
+                        <TableRow key={row}>
```
Comment (legendarykamal) [inline ui/components/MesherySMIResults.js:230]: If the results are dynamic which in this case it would be , here key should be `key={row.uniqueID}`

**Review by legendarykamal — CHANGES_REQUESTED** [review]

File: ui/components/MesherySMIResults.js (hunk 1 above)
Comment (sam1code) [reply ui/components/MesherySMIResults.js:230]: Ok i'm pushing the change

**Comment by sam1code** [comment]
Pushed the changes according to you instruction Please check, thank you.

**Comment by leecalcote** [comment]
> If possible can we have unique IDs in all these keys if exists? Rest, LGTM

@sam1code, feedback ^^

**Comment by sam1code** [comment]
yes I'm updating the pr in some times!

**Comment by sam1code** [comment]
Now I think this is done
I've updated the uniqueid as key.

**Comment by leecalcote** [comment]
Yay, @sam1code! 🎈

**Comment by sam1code** [comment]
😍

**Pull request merged**